app/src/main/java/cm/aptoide/pt/appview/UserPreferencesPersister.java

```diff
@@ -0,0 +1,26 @@
+package cm.aptoide.pt.appview;
+
+import android.content.SharedPreferences;
+
+/**
+ * Created by filipegoncalves on 5/7/18.
+ */
+
+public class UserPreferencesPersister {
+
+  private SharedPreferences sharedPreferences;
+
+  public UserPreferencesPersister(SharedPreferences sharedPreferences) {
+    this.sharedPreferences = sharedPreferences;
+  }
+
+  public void save(String key, int value) {
+    sharedPreferences.edit()
+        .putInt(key, value)
+        .apply();
+  }
+
+  public int get(String key, int defaultValue) {
```

Comment:
should the default value be passed as parameter? default value should probably be hidden inside the method.